### DISCUSSION TITLE
To initialize filter values in params when reloaded from a specific list page

### DISCUSSION THREAD

**Discussion by KoYoungWock** [refinedev/refine#5224] [Q&A]
I'm implementing a logic that resets the filter value when I reload from a specific list page as shown below. However, even if you write the code as shown below, there is a problem of navigation to url including filter value entered before. How should I solve this problem?

```
import { useTable } from '@refinedev/core'

const { setFilters } =  useTable<IUserInfo, HttpError>({
    resource: 'users',
    dataProviderName: 'users',
    syncWithLocation: true,
})

const preventClose = useCallback(() => {
    setFilters([])
}, [])

useEffect(() => {
    (() => {
        window.addEventListener("beforeunload", preventClose);    
    })();

    return () => {
        window.removeEventListener("beforeunload", preventClose);
    };
},[]);
```

**Comment by BatuhanW** [comment]
Hello @KoYoungWock couldn't understand the issue from the initial message. Could you provide us a reproducible example and explain with more details? Thanks.
